### ISSUE TITLE
Sequence format

### ISSUE THREAD

**Issue by MohammedFCIS** [Dainanahan/IDG-DREAM-Drug-Kinase-Binding-Prediction-Challenge#7]
http://137.132.97.65/cgi-bin/profeat2016/protein/profnew.cgi
has the option to upload sequence in RAW or FASTA, so why we need to convert it to RAW?
--

**Comment by alizat** [comment]
This was a mistake on my part. We actually do NOT need to convert FASTA to RAW. I previously assumed that it needed to be converted to RAW because that is what I uploaded during my work in the PhD thesis. I was not aware (or I forgot) that PROFEAT also supports FASTA.

**Comment by MohammedFCIS** [comment]
still using RAW is better for later processing

**Comment by alizat** [comment]
Just for the record, it seems that using FASTA as input to the PROFEAT server seems to be "more likely to work properly". There were a few times where I tried using RAW sequences as input to PROFEAT but only to receive an empty features file. In these cases, when I tried converting (manually) the RAW sequences into FASTA format, the features were successfully generated.

I believe this is also why I used FASTA files as input when I used this same server in the PhD. Just something to keep in mind.
